### PR TITLE
Updated Coralogix screenshots and wording for Postman steps

### PR DESCRIPTION
**Pull request by jerry-li-dev**
Updated screenshots and wording for Postman steps.

### CODE REVIEW

**Comment by jerry-li-dev** [comment]
@shashankawasthi88 Could you please help me confirm if the steps for Coralogix software side are still correct? I do not have an account to test it.

File: src/pages/docs/integrations/available-integrations/coralogix.md [hunk 1]
```diff
@@ -31,33 +31,27 @@ Navigate to "Send your logs" tab and copy the private key for later use, as illu
...
-   * Optionally indicate if you want to send collection or environment details for each run.
+    * Select the monitor whose data you would like to send to Coralogix.
+    * Enter a Coralogix private key.
+    * Optionally indicate if you want to send collection or environment details for each run.
 
-1. Click **Add Integration**.
+    Select **Add Integration** to save the configuration.
 
-   Once you click _Add Integration_, the following screen appears to indicate the integration is configured:
+1. Select the name of the collection added to this integration to see review the monitoring metrics within Postman.
```
Comment (shashankawasthi88) [inline src/pages/docs/integrations/available-integrations/coralogix.md:50]: @jerry-li-dev I dint get the reference to the collection? The Integration is configured over a monitor (which inturn is on Collection). We don't directly show this reference on the Postman Integrations UI as such.

Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/coralogix.md:50]: Ok, I will remove this section from the page.

File: src/pages/docs/integrations/available-integrations/coralogix.md [hunk 2]
```diff
@@ -31,33 +31,21 @@ Navigate to "Send your logs" tab and copy the private key for later use, as illu
 
 ## Configuring Coralogix Integration
 
-1. In the [Integrations](https://go.postman.co/workspaces) tab for your workspace, select Coralogix from the list of third party integrations.
+1. From the [Integrations](https://go.postman.co/integrations/browse?category=all) page, search and select Coralogix from the list of integrations.
 
-   [![coralogix integrations page](https://assets.postman.com/postman-docs/coralogix_viewdetails.png)](https://assets.postman.com/postman-docs/coralogix_viewdetails.png)
+   ![coralogix integrations page](https://assets.postman.com/postman-docs/coralogix-search.jpg)
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/coralogix.md:36]: Let's remove the extra white space on the right and add a width so it doesn't expand to the full screen

**Review by claireriley — APPROVED** [review]
Looks great @jerry-li-dev ! Just one suggestion to check out

**Pull request merged**